src/test/java/io/citrine/jcc/search/pif/query/PifSystemReturningQueryIT.java

```diff
@@ -148,6 +148,51 @@ public void testAnalysis() throws IOException {
         Assert.assertEquals(1, ((CategoricalAnalysisResult) pifSearchResult.getAnalysis("name")).bucketsLength());
     }
 
+    /**
+     * Test that the includeNumeric option is used when running an analysis.
+     *
+     * @throws IOException if thrown while making search requests.
+     */
+    @Test
+    public void testIncludeNumeric() throws IOException {
+
+        // To reduce the time that the query is going to run on the server, this test has two parts. First, query
+        // for a PIF from any dataset.
+        final PifSearchResult pifSearchResult = this.client.search(new PifSystemReturningQuery()
+                .setSize(1)
+                .addQuery(new DataQuery()
+                        .addSystem(new PifSystemQuery()
+                                .addUid(new Filter().setExists(true)))));
+        final String uid = pifSearchResult.getHits(0).getId().split("/", 3)[2];
+
+        // Filter on the UID of the PIF that was returned then run a categorical analysis on the length of the
+        // system, which should always be 1. Since numeric values are not included in categorical analyses, this
+        // should not return any buckets.
+        final PifSearchResult withoutNumericResult = this.client.search(new PifSystemReturningQuery()
+                .setSize(0)
+                .addQuery(new DataQuery()
+                        .addSystem(new PifSystemQuery()
+                                .addUid(new Filter()
+                                        .setEqual(uid))
+                                .addLength(new FieldQuery()
```

Comment:
What is the length of the system?

Reply:
During import, every object in the PIF record gets enriched with a length and offset value. For objects not in a list, that will always be 1 and 0, respectively. For objects in a list, it might look something like `[{length: 2, offset: 0}, {length: 2, offset: 1}]`.

Reply:
This just lets us do things like query for records with at least 2 subsystems or query for records where the first author's last name is Michel.

Reply:
Given its relative obscurity, it just seems like a bit of an odd choice for a tests.  I understand that you picked it because it is automatically added to every PIF, but it might be nice to follow a more common codepath.  Just a comment though; I can merge if you nack.

Reply:
I used length for this test because it is _always_ numeric. So it makes it easy to get a record from any site that we might be testing against that would behave in a predictable way.

Reply:
I'd prefer to keep it the way it is. The test is going to get pretty complicated otherwise.

Reply:
Also, this test is really just to ensure that includeNumeric is being passed to the server correctly. We test includeNumeric against a lot of fields in the server-side library tests.